announcing-xata-sdk.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Announcing the release of the Xata Go SDK'
+description: 'We are excited to announce the initial alpha version of the Xata Go SDK.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/announcing-xata-sdk-cover.jpg
+  alt: Xata Go SDK
+author: Philip Krauss
+date: 12-06-2023
+published: true
+slug: announcing-xata-go-sdk
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/announcing-xata-sdk-cover.jpg
+---
+
+Earlier this year we shared a [community spotlight](https://xata.io/blog/community-spotlight-xata-go-sdk) focused on a community contributed Go SDK, [xata-go](https://github.com/kerdokurs/xata-go). [Kerdo Kurs](https://www.linkedin.com/in/kerdokurs/), the developer behind this project, was a huge inspiration for us committing to an official Go SDK. Today, we're thrilled to announce the first release of our Xata Go SDK, and to open the doors to the growing Go community 🎉
```

Comment:
I don't know how to get away from using the word community twice in this sentence haha. Any ideas @joan-ing ?

Reply:
addressed via https://github.com/xataio/mdx-blog/pull/165/commits/cbd9730c2d849fa0ddf183310a42ec55909eb8ec